pkg/interface/src/views/themes/light.ts

```diff
@@ -162,7 +162,7 @@ const theme = {
   ],
   // z-index
   zIndices: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
-  breakpoints: ["768px", "1024px", "1440px", "2200px"],
+  breakpoints: ["550px", "750px", "960px"],
```

Comment:
Do these breakpoints follow the ones you added to #3236 ?

Reply:
yep